src/sass/claims-status.scss

```diff
@@ -138,80 +133,13 @@
   }
 }
 
-.claims-evidence {
-  border-bottom: 1px solid $color-gray-light;
-  padding: 1em 0;
-  &.row {
-    margin: 0 0;
-  }
-  .claims-evidence-date {
-    font-weight: bold;
-    padding-right: 10px;
-  }
-  &:last-child {
-    border: none;
-    padding-bottom: 0;
-  }
-}
-
-.completion-container {
-  margin-left: 8px;
-}
-
-.claim-completion-estimation {
-  p {
-    margin: 0;
-  }
-  .date-estimation {
-    display: inline-block;
-    font-weight: bold;
-    margin-right: 10px;
-  }
-
-}
-
-.claim-decision-is-ready,
 .you-have-no-claims {
   h4, p {
```

Comment:
Is this style being used (`.you-have-no-claims`)? If not, it would be a good opportunity to remove some additional tag name selectors.

Reply:
I think it's used on the claim list page, if you don't have any claims.